app/assets/javascripts/initializers/initializeAllChatButtons.js

```diff
@@ -1,7 +1,14 @@
-function initModal() {
-  var modal = document.querySelector('.modal');
-  modal.querySelector('.close-modal').addEventListener('click', toggleModal);
-  modal.querySelector('.overlay').addEventListener('click', toggleModal);
+'use strict';
+
+function showChatModal(modal) {
```

Comment:
This code can be reworked, but I'm just focusing on fixing lint issues at the moment.